scripts/crowbar-testbuild.yaml

```diff
@@ -11,6 +11,10 @@ branches:
     bs_project:  Devel:Cloud:7:Staging
     bs_repo:     SLE_12_SP2
     cloudsource: develcloud7
+  master:
```

Comment:
Would be the second step, first we need to stabilize Cloud 8 to be able to have a gating and testbuild for master/Cloud8.